### ISSUE TITLE
Definición estrato con la estación

### ISSUE THREAD

**Issue by Franvgls** [Eucrow/NewCamp#12]
Observo que el estrato se mete al principio de los datos del lance, y eso no está mal, pero si es así debería servir tb para control de calidad, de no poder meter datos que estén fuera del estrato (profundidad, latitud y longitud)

**Comment by Eucrow** [comment]
Ese campo 'Stratum' está así temporalmente. La idea es que cuando se implementen los estratos en la configuración de la campaña ese campo se mostrará pero sólo a nivel informativo.

**Comment by Franvgls** [comment]
Perfecto entonces. Pero entonces entiendo que sólo se mostrará una vez terminado, o determinado por la profundidad y posición. Aunque tb hay estaciones que se considerarán fuera de estrato por ser lances especiales/extra.

**Comment by Eucrow** [comment]
Sí, según la profundidad y la posición mostrará el estrato. Las que no entren en el estrato, pues pueden aparecer vacío o con una etiqueta de "especial" o como veais.

**Comment by Franvgls** [comment]
Yo creo que especial y nulo completan las posibilidades.
Gracias
